### PR TITLE
Added new open source iOS game Alienblast 

### PR DESCRIPTION
**Pull request by etamity**
0. [x] Project URL:
1. [x] Update contents.json instead of README 
2. [x] One project per pull request
4. [x] Avoid iOS or open-source in description as it is assumed
5. [ ] Use this commit title format if applicable: Add app-name by @github-username
6. [ ] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -1136,7 +1136,13 @@
       "tags": [
         "swift"
       ]
-    },
+    }, {
+    "title": "Alienblast",
+    "category": "games",
+    "description": "Cute and sweet game developed by Swift, Coco2d and Spritebuilder.",
+    "source": "https://github.com/etamity/AlienBlast",
+    "tags": ["iOS","swift"]
```
Comment (dkhamsing) [inline contents.json]: tags should be just `["swift"]`

File: contents.json [hunk 2]
```diff
@@ -1136,7 +1136,13 @@
       "tags": [
         "swift"
       ]
-    },
+    }, {
+    "title": "Alienblast",
+    "category": "games",
+    "description": "Cute and sweet game developed by Swift, Coco2d and Spritebuilder.",
```
Comment (dkhamsing) [inline contents.json]: I think it's `Cocos2d`

**Comment by dkhamsing** [comment]
@etamity 
can you include license info, thanks

**Comment by etamity** [comment]
@dkhamsing  I have added LICENSE file to the project, thanks for reminding.

**Comment by dkhamsing** [comment]
hehe
I mean update the pull request to include license ;)

**Comment by dkhamsing** [comment]
will fix issues myself

**Pull request merged**